manageiq-providers-hawkular.gemspec

```diff
@@ -13,7 +13,7 @@ Gem::Specification.new do |s|
 
   s.files = Dir["{app,config,lib}/**/*"]
 
-  s.add_runtime_dependency "hawkular-client", "~> 3.0.2"
+  s.add_runtime_dependency "hawkular-client", "~> 4.0.0"
```

Comment:
Since this is a shared dependency with manageiq-providers-kubernetes (and thus, manageiq-providers-openshift as well), please verify that a major bump will not break them.  https://github.com/ManageIQ/manageiq-providers-kubernetes/blob/e0e4b5f427aec60e28a15cafd07345f48fcdee25/manageiq-providers-kubernetes.gemspec#L16